.github/workflows/build.yml

```diff
@@ -270,6 +270,12 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
+      -
+        name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4.3.0
```

Comment:
please use the commit hash and add the version as a comment behind it.

Reply:
I've made this change